CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added `network` field to `compute_target_instance` ([#7189](https://github.com/hashicorp/terraform-provider-google/pull/7189))
+* compute: added `storage_locations` field to `google_compute_snapshot` ([#7201](https://github.com/hashicorp/terraform-provider-google/pull/7201))
```

Comment:
this PR looks like it also does a few other things, can those get changelog notes too?